forms/gridfield/GridFieldAddExistingAutocompleter.php

```diff
@@ -232,7 +232,12 @@ public function doSearch($gridField, $request) {
 		$originalSourceFileComments = Config::inst()->get('SSViewer', 'source_file_comments');
 		Config::inst()->update('SSViewer', 'source_file_comments', false);
 		foreach($results as $result) {
-			$json[$result->ID] = html_entity_decode(SSViewer::fromString($this->resultsFormat)->process($result));
+			$title = html_entity_decode(SSViewer::fromString($this->resultsFormat)->process($result));
+			$json[] = array(
+				"label" => $title,
```

Comment:
Sorry to nitpick, but these should really be single quoted to stay consistent with the rest of the file (and SS in general)